src/pages/Auth/Login.jsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react'
+import { Helmet } from 'react-helmet-async'
+import { useForm } from 'react-hook-form'
+import InputField from 'src/components/InputField/InputField'
+import * as yup from 'yup'
+import { yupResolver } from '@hookform/resolvers/yup'
+import { useDispatch } from 'react-redux'
+import { login } from './auth.slice'
+import { unwrapResult } from '@reduxjs/toolkit'
+import { useNavigate } from 'react-router-dom'
+import { path } from 'src/constants/path'
+import Button from 'src/components/Button/Button'
+
+export default function Login() {
+  const dispatch = useDispatch()
+  const navigate = useNavigate()
+
+  const schema = yup.object().shape({
+    email: yup
+      .string()
+      .required('Vui lòng nhập địa chỉ email')
+      .email('Vui lòng nhập email hợp lệ'),
+    password: yup
+      .string()
+      .required('Vui lòng nhập mật khẩu')
+      .min(6, 'Vui lòng nhập 6-20 kí tự')
+      .max(20, 'Vui lòng nhập 6-20 kí tự')
+  })
+  const form = useForm({
+    defaultValues: {
+      email: '',
+      password: ''
+    },
+    resolver: yupResolver(schema)
+  })
+
+  const handleSubmit = async data => {
+    const body = {
+      email: data.email,
+      password: data.password
+    }
+    try {
+      const res = await dispatch(login(body))
```

Comment:
```suggestion
      const res = await dispatch(login(data))
```
như này là được rồi em

Reply:
cái này bị trùng với biến data nên em đặt body anh ạ